dvc/command/remote.py

```diff
@@ -39,6 +44,13 @@ def run(self):
             )
 
         section = Config.SECTION_REMOTE_FMT.format(self.args.name)
+
+        try:
+            self._prevent_config_overwriting(section)
+        except DvcException as exc:
```

Comment:
There is no point in raising an exception just to catch it right here. You could simply print the error and `return 1`.

Reply:
Ok, got it!) Just decided to implement it in the same manner as previous pr - https://github.com/iterative/dvc/pull/1781, will fix it